brails/workflow/Images.py

```diff
@@ -86,12 +101,10 @@ def download(urls):
 def getGoogleImages(footprints=None, GoogleMapAPIKey='',imageTypes=['StreetView','TopView'],imgDir='',ncpu=2,fov=60,pitch=0,reDownloadImgs=False):
 
     if footprints is None:
```

Comment:
It could be better to do away with this `if` block altogether and just remove the default value in the function declaration.